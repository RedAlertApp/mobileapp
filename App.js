import React from "react"
import Ui from "./src/Ui.js"
import Reports from "./src/Reports.js"
import { Provider } from "react-redux"
import store from "./src/store.js"
import ScrollableTabView from "react-native-scrollable-tab-view"
import CustomTabBar from "./src/CustomTabBar"

class App extends React.Component {
  render() {
    return (
      <Provider store={store}>
        <ScrollableTabView
          renderTabBar={() => <CustomTabBar />}
          initialPage={1}
          tabBarPosition="bottom"
          style={{ backgroundColor: "#FAFAFA" }}
        >
          <Ui tabLabel="Profil" />
          <Ui tabLabel="Zgłoś problem" />
          <Reports tabLabel="Lista zgłoszeń" />
        </ScrollableTabView>
      </Provider>
    )
  }
}
export default App
