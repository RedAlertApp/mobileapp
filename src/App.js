import React from "react"
import Ui from "./Ui.js"
import Reports from "./Reports.js"
import { Provider } from "react-redux"
import store from "./store.js"
import ScrollableTabView from "react-native-scrollable-tab-view"
import CustomTabBar from "./CustomTabBar"

class App extends React.Component {
  render() {
    return (
      <Provider store={store}>
        <ScrollableTabView
          renderTabBar={() => <CustomTabBar />}
          initialPage={1}
          tabBarPosition="bottom"
          style={{ backgroundColor: "#FAFAFA" }}
        >
          <Ui tabLabel="Profil" />
          <Ui tabLabel="Zgłoś problem" />
          <Reports tabLabel="Lista zgłoszeń" />
        </ScrollableTabView>
      </Provider>
    )
  }
}
export default App
