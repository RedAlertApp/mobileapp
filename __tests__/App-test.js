import "react-native"
import React from "react"
import App from "../App"
import renderer from "react-test-renderer"

describe("App component", () => {
  jest.useFakeTimers()

  it("renders without crashing", async () => {
    const tree = renderer.create(<App />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
