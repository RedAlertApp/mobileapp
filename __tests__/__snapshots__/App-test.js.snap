// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders without crashing 1`] = `
<View
  onLayout={[Function]}
  style={
    Array [
      Object {
        "flex": 1,
      },
      Object {
        "backgroundColor": "#FAFAFA",
      },
    ]
  }
/>
`;
